chroma-agent/chroma-agent.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=IML Agent Daemon
+After=network.target
+
+[Service]
+Type=simple
+ExecStart=/usr/bin/chroma-agent-daemon
```

Comment:
any reason why in /usr/bin/ rather than /usr/lib64/ to be consistent with our other daemon components? we are also not really running it as a daemon with this approach, so we may not want to call it a Daemon anyway, really is just a standard systemd service!

Reply:
yeah, I think that's coming from the `setup.py` invocation in the spec file. 

I haven't moved it to a new location btw, just calling the same thing that was in the init script.